spec/membership_comparison_spec.rb

```diff
@@ -40,6 +40,24 @@
     specify { expect(comparison.problems).to be_empty }
   end
 
+  context 'single existing P39, bare statements' do
```

Comment:
this claims there's only a single existing P39, but the `existing` below shows two entries…